addons/mass_mailing/models/utm.py

```diff
@@ -10,8 +10,10 @@ class UtmCampaign(models.Model):
     mailing_mail_ids = fields.One2many(
         'mailing.mailing', 'campaign_id',
         domain=[('mailing_type', '=', 'mail')],
-        string='Mass Mailings')
-    mailing_mail_count = fields.Integer('Number of Mass Mailing', compute="_compute_mailing_mail_count")
+        string='Mass Mailings',
+        groups="mass_mailing.group_mass_mailing_user")
+    mailing_mail_count = fields.Integer('Number of Mass Mailing', compute="_compute_mailing_mail_count", groups="mass_mailing.group_mass_mailing_user")
+    is_campaign_activated = fields.Boolean(compute="_compute_is_campaign_activated")
```

Comment:
Naming: "is_mailing_campaign_activated" ? Because "is_campaign_activated" on campaign model is a bit misleading :)